.github/workflows/scheduled-weekly-new-nodejs-lts-check.yml

```diff
@@ -0,0 +1,47 @@
+name: Weekly new Node.js LTS check
+on:
+  workflow_dispatch:
+
+  # Run on every Sunday to check if a new Node.js LTS that we don't support is released, and create an Issue if this is the case
+  schedule:
+    - cron: "0 4 * * 0"
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  load-nodejs-supported-versions:
+    uses: ./.github/workflows/reusable-load-nodejs-supported-versions.yml
+
+  lts-check:
+    needs: load-nodejs-supported-versions
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get latest Node.js LTS versions
+        run: |
+          curl -s https://nodejs.org/dist/index.json | jq '[.[] | select(.lts != false)] | .[0]' > latest-lts.json
+          CURRENT_LTS=$(jq -r '.version' latest-lts.json | sed 's/v//')
+          echo "Latest LTS version: $CURRENT_LTS"
```

Comment:
For readability, I would suggest using `${CURRENT_LTS}` rather than a naked variable.